components/dashboard/src/SwitchToPAYG.tsx

```diff
@@ -259,55 +259,8 @@ function SwitchToPAYG() {
                         }
                     }
                     if (!cancelled && !attributionId) {
-                        setPageState((s) => {
-                            const attributionId = s.attributionId || derivedAttributionId;
-                            return { ...s, attributionId, old };
-                        });
-                    }
-                })().catch(console.error);
-
-                return () => {
-                    cancelled = true;
-                };
-            }
-
-            case "wait-for-signup": {
```

Comment:
consolidated in [subscribeToStripe](https://github.com/gitpod-io/gitpod/pull/16822/files#diff-ed184a7f0527f0afb690cf6358ef432cb908179e7ff74dc26691375cb92589b6R648)